Makefile.am

```diff
@@ -50,3 +50,11 @@ dist_noinst_DATA = \
 	AUTHORS COPYING README \
 	example.c \
 	testinput.c
+
+if COMPILE_EXAMPLE
+check_PROGRAMS = example
+example_SOURCES = example.c
+example_LDFLAGS = -std=gnu99 -ggdb -O0
```

Comment:
configure.ac should take care of the correct -std=gnu99 setting, and the other two are user-specified.